src/EnergyPlus/HeatPumpWaterToWaterSimple.cc

```diff
@@ -96,31 +95,24 @@ namespace EnergyPlus::HeatPumpWaterToWaterSimple {
 // M.S. Thesis, Department of Mechanical and Aerospace Engineering,
 // Oklahoma State University. (downloadable from http://www.hvac.okstate.edu/)
 
-// OTHER NOTES: none
-
-// USE STATEMENTS:
-// Use statements for data only modules
-// Using/Aliasing
-using namespace DataLoopNode;
-
 // MODULE PARAMETER DEFINITIONS
 std::string const HPEqFitHeating("HeatPump:WatertoWater:EquationFit:Heating");
```

Comment:
Can these things be made `constexpr std::string_view`? Or is the epJSON infrastructure not ready to take those yet?

Reply:
VerifyUniqueInterObjectName takes a string